backend/app/chat/config.py

```diff
@@ -1,3 +1,4 @@
 import os
 
-OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
```

Comment:
QQ: If we set LiteLLM to `gpt-4o` will that just read from local env variables (scanning for `OPENAI_API_KEY` I presume?)

Reply:
N.B. Things still worked fine for me! Just asking out of curiosity more than anything!

Reply:
Yes exactly. We actually don't use `OPENAI_API_KEY` anywhere explicitly. For example, we don't even pass it to litellm when we call it. It just looks it up if model is an openai one. 

So we don't need to have it here.

One thing we should do though is have a healthcheck endpoint that checks if env vars are setup correctly and we can access the LLM, we can contact postgres and redis, etc. I'l raise a ticket for that

Reply:
I think that would be great at sorting a lot of setup issues out :)

Reply:
Created [AFHC-44](https://lastmilehealth.atlassian.net/browse/AFHC-44?atlOrigin=eyJpIjoiY2RmMDJiMjM1NWZkNDMzNTk2ZDAzOGI5MmY4YTU3MWUiLCJwIjoiaiJ9)